src/test/java/org/apache/commons/lang3/StringUtilsTest.java

```diff
@@ -1623,13 +1622,9 @@ public void testReCapitalize() {
     public void testRemove_char() {
         // StringUtils.remove(null, *)       = null
         assertNull(StringUtils.remove(null, 'a'));
-        assertNull(StringUtils.remove(null, 'a'));
-        assertNull(StringUtils.remove(null, 'a'));
 
         // StringUtils.remove("", *)          = ""
         assertEquals("", StringUtils.remove("", 'a'));
-        assertEquals("", StringUtils.remove("", 'a'));
-        assertEquals("", StringUtils.remove("", 'a'));
```

Comment:
This method, `testRemove_char`, appears to have been based off the `testRemove_String`. You are correct these three statements are duplicated and can be removed. But in `testRemove_String` there's one test case that must be added here as well:

```
assertNull(StringUtils.remove(null, null));
&
assertEquals("", StringUtils.remove("", null));
```

Can you add them, please, @shalk ?

Reply:
done